runners/spark/src/main/java/org/apache/beam/runners/spark/translation/streaming/StreamingTransformTranslator.java

```diff
@@ -115,13 +121,35 @@ public void evaluate(Read.Unbounded<T> transform, EvaluationContext context) {
     };
   }
 
-  private static <T> TransformEvaluator<CreateStream.QueuedValues<T>> createFromQueue() {
-    return new TransformEvaluator<CreateStream.QueuedValues<T>>() {
+  private static <T> TransformEvaluator<CreateStream<T>> createFromQueue() {
```

Comment:
Well, it could be a class with a bunch of inner (static) classes, but that won't be too different.
On the other hand, I'm not sure It'd be better to have another bunch of files..
Maybe once we separate streaming/batch translators entirely it'd be a bit better.

I tried making this easy to review by committing those changes as part of a "following refactor" commit meaning that it's just adjusting the weldings so that tests (passing) should give the reviewer a good sense of wether it's OK or not.